cmd/goQuery/cmd/root.go

```diff
@@ -32,13 +32,16 @@ var cfgFile string
 var supportedCmds = "{QUERY TYPE|COLUMNS|examples|list|version}"
 
 var rootCmd = &cobra.Command{
-	Use:           "goQuery -i <interfaces> QUERY TYPE",
-	Short:         helpBase,
-	Long:          helpBaseLong,
-	RunE:          entrypoint,
-	Args:          validatePositionalArgs,
-	SilenceUsage:  true,
-	SilenceErrors: true,
+	Use:   "goQuery -i <interfaces> QUERY TYPE",
+	Short: helpBase,
+	Long:  helpBaseLong,
+	// we want to make sure that every command can be profiled
+	PersistentPreRunE:  initProfiling,
```

Comment:
The use of `PersistentPreRun` will make sure that this set up code is called before _every_ command, not just the query commands.

This is relevant so we also get profiles if `list` is called.

Reply:
I see, thanks for cross-checking / proposing the change! Good to go then?